internal/controller/datadogagentinternal/component/agent/default.go

```diff
@@ -334,9 +334,7 @@ func agentImage() string {
 }
 
 func otelAgentImage() string {
```

Comment:
This [PR](https://github.com/DataDog/datadog-operator/pull/1916/files#diff-99f3b0c011db2b1425d672097cb8258747a30d66cf70e79275681ffb992bbcd4)  updated this function in DatadogAgent controller in a bit different way so better to follow same approach.